src/cablab/providers/air_temperature.py

```diff
@@ -29,7 +27,7 @@ def get_variable_descriptors(self):
         return {
             VAR_NAME: {
                 'data_type': numpy.float32,
-                'fill_value': -32767,
+                'fill_value': FILL_VALUE,
                 'units': 'K',
                 'long_name': '2 metre temperature',
                 'scale_factor': 0.0019718202938428923,
```

Comment:
Do you mean that the data conversion should be handled by the provider and then the offset and scale values information is removed on the resulting netCDF file (as well as the change of unit from K to C)?